.env

```diff
@@ -0,0 +1,7 @@
+REACT_APP_API_KEY=AIzaSyDmmvSyGcCLU_gBV1LPoV6nh6gPCPfN68M
```

Comment:
never ever ever push the .env file. Always create a .gitignore file and add it in there